### DISCUSSION TITLE
About the issue of Windows Forms Demo in the client

### DISCUSSION THREAD

**Discussion by layfolk233** [micjahn/ZXing.Net#543] [Q&A]
In the Windows Forms Demo software, in the WebCam option, when connecting a USB camera and clicking Decode, the picture box appears black with no images. Have you encountered this situation?
![屏幕截图 2024-01-12 151037](https://github.com/micjahn/ZXing.Net/assets/73208956/92c68c35-9958-4728-b79e-e9a09ec24dba)

I ensure that the camera is functioning properly and images can be seen in the built-in camera software on Windows.
![屏幕截图 2024-01-12 151214](https://github.com/micjahn/ZXing.Net/assets/73208956/c91dc94f-6c59-452c-be49-9f6386f7440d)

**Comment by layfolk233** [comment]
I tried versions 0.16.9, 0.16.8, and 0.15.0, all of which yielded the same results

**Comment by micjahn** [comment]
I fear I can't help here.
Do you have a second camera attached to your computer?

**Reply by layfolk233** [reply]
Thank you for your reply!

Then I can check the source code in the project and find the reason myself. As long as the demo function is normal, I should be able to find a solution quickly.

(I have tried using two cameras, USB interface UVC protocol, plugged into the computer, and both can be used normally in the built-in "Camera" software of Windows 10)

**Reply by layfolk233** [reply]
Could you try to run the "WebCam" function of "WindowsFormsDemo" and take a screenshot to show me what the correct effect is? What will be displayed in the black box?
